pkgs/sketch_pad/lib/flutter_samples.dart

```diff
@@ -0,0 +1,32 @@
+// Copyright (c) 2024, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+import 'package:http/http.dart' as http;
+
+class FlutterSampleLoader {
+  final http.Client client = http.Client();
+
+  Future<String> loadFlutterSample({
+    required String sampleId,
+    String? channel,
+  }) async {
+    // There are only two hosted versions of the docs: master/main and stable.
+    final sampleUrl = (channel == 'master' || channel == 'main')
+        ? 'https://main-api.flutter.dev/snippets/$sampleId.dart'
+        : 'https://api.flutter.dev/snippets/$sampleId.dart';
```

Comment:
nit: use the new switch expression syntax instead of a ternary?

Reply:
done!